bootstrap_cfn/fab_tasks.py

```diff
@@ -342,6 +343,10 @@ def cfn_create():
     try:
         stack = cfn.create(stack_name, cfn_config.process())
     except:
+        # cleanup ssl certificates if any
+        if 'ssl' in cfn_config.data:
+            print red("Deleting SSL certificates from stack")
+            iam.delete_ssl_certificate(cfn_config.ssl(), stack_name)
```

Comment:
I don't think it's necessary nor worth to handle an exception at this level:
1) delete_ssl_certificate is a function in iam.py that always returns True and doesn't do any check
```
    def delete_ssl_certificate(self, ssl_config, stack_name):
        for cert_name, ssl_data in ssl_config.items():
            self.delete_certificate(cert_name,
                                    stack_name,
                                    ssl_data)
        return True
```

2) running some tests and checking the [documentation](http://docs.aws.amazon.com/IAM/latest/APIReference/API_DeleteServerCertificate.html), the behaviour of the AWS API is to return an error:
```
ERROR:boto:404 Not Found
ERROR:boto:<ErrorResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
  <Error>
    <Type>Sender</Type>
    <Code>NoSuchEntity</Code>
    <Message>The Server Certificate with name opstest1-cert-crimebillingonline-opstest1-26ba772f1 cannot be found.</Message>
  </Error>
  <RequestId>322b4f8f-13ac-11e5-931d-11a40828e9fc</RequestId>
</ErrorResponse>
```

3) delete_certificate in iam.py already handles exceptions locally:
```
    def delete_certificate(self, cert_name, stack_name, ssl_data):
        """
        Delete a certificate from AWS

        Args:
                cert_name(string): The name of the certificate entry to look up
                stack_name(string): The name of the stack
                ssl_data(dictionary): The configuration data for this
                    certificate entry

        Returns:
            success(bool): True if a certificate is deleted, False otherwise
        """
        cert_id = "{0}-{1}".format(cert_name, stack_name)
        # Try to delete cert, but handle any problems on
        # individual deletes and
        # continue to delete other certs
        try:
            if self.get_remote_certificate(cert_name,
                                           stack_name):
                self.conn_iam.delete_server_cert(cert_id)
                logging.info("IAM::delete_certificate: "
                             "Deleting certificate '%s'.."
                             % (cert_name))
                return True
            else:
                logging.info("IAM::delete_certificate: "
                             "Certificate '%s' does not exist, "
                             "not deleting." % (cert_name))
                return False
        except AWSQueryConnection.ResponseError as error:
            logging.warn("IAM::delete_certificate: "
                         "Could not find expected certificate '%s': "
                         "Error %s - %s" % (cert_id,
                                            error.status,
                                            error.reason))
            return False

        return False
```

Based on the above I would like to resubmit the PR as-is for review.